_includes/navigation.html

```diff
@@ -1,9 +1,12 @@
 <ul class="top-level">
   <li>
     {% if page.section == 'discovery' %}
-      {% include discovery.html %}}
+      {% include discovery.html %}
+    {% elsif page.section == 'alpha' %}
+        {% include alpha.html %}
     {% else %}
       <a href="{{ site.baseurl }}/discovery">Discovery</a>
+      <a href="{{ site.baseurl }}/alpha">Alpha</a>
```

Comment:
Could you remove this from the main navigation for now - whilst the guide is still in review?